pages/governance/proposal/index.governance.tsx

```diff
@@ -19,34 +17,27 @@ export default function DynamicProposal() {
   const [proposal, setProposal] = useState<CustomProposalType>();
   const [ipfs, setIpfs] = useState<IpfsType>();
 
-  async function updateProposal() {
-    const { values, ...rest } = await governanceContract.getProposal({ proposalId: id });
-    setProposal(await enhanceProposalWithTimes(rest));
+  async function initialize() {
```

Comment:
this was a bit unnecessary complex as the proposal and ipfs fetching where made independent although they belong together.